homeassistant/helpers/selector.py

```diff
@@ -471,15 +471,18 @@ def __call__(self, data: Any) -> Any:
 
 @SELECTORS.register("select")
 class SelectSelector(Selector):
-    """Selector for an single-choice input select."""
+    """Selector for an single or multi-choice input select."""
 
     selector_type = "select"
 
     CONFIG_SCHEMA = vol.Schema(
         {
             vol.Required("options"): vol.All(
                 vol.Any([str], [select_option]), vol.Length(min=1)
```

Comment:
We need to allow an empty options list for custom options:
```suggestion
                vol.Any([str], [select_option])
```